### PR TITLE
chore(web): add story text block

### PR DESCRIPTION
**Pull request by airslice**
# Overview

This PR adds story text block.

RichTextEditor is implemented as a third part library component using lexical.

## What I've done

- Add `RichTextEditor`
- Add story text block
- Change story block settings panel as relative element (as design).
- Change story block basic structure from using children to use render function.

## What I haven't done

- Didn't migrate the RichTextEditor to reearth as a native component.

### CODE REVIEW

**Review by nina992 — APPROVED** [review]

File: web/src/beta/features/Editor/StoryPanel/Block/builtin/Text/index.tsx [hunk 1]
```diff
@@ -15,19 +17,31 @@ const TextBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
     [block?.property?.items],
   );
 
+  // Text block is very special, it will not edit values with field components
+  // from the common editor panel, but manage it by itself directly.
```
Comment (KaWaite) [inline web/src/beta/features/Editor/StoryPanel/Block/builtin/Text/index.tsx:21]: Can you move this above the component's declaration? Seems related to the component itself, but is above the usePRopertyValueUpdate but seems unrelated

**Review by KaWaite — APPROVED** [review]

File: web/src/beta/features/Editor/StoryPanel/Block/builtin/Text/index.tsx (hunk 1 above)
Comment (airslice) [reply web/src/beta/features/Editor/StoryPanel/Block/builtin/Text/index.tsx]: Updated! I'll close this conversation since you have approved. Thank you!
